lua/gitblame/init.lua

```diff
@@ -680,19 +687,19 @@ local create_cmds = function()
 end
 
 ---@class SetupOptions
----@field enabled boolean
----@field message_template string
----@field date_format string
----@field message_when_not_committed string
----@field highlight_group string
----@field gitblame_set_extmark_options object @See :h nvim_buf_set_extmark() to check what you can pass here
----@field display_virtual_text boolean
----@field ignored_filetypes string[]
----@field delay number @Visual delay for displaying virtual text
+---@field enabled? boolean
```

Comment:
All the setup options were required so I marked them all optional. `type?` is the same as `type|nil` https://luals.github.io/wiki/annotations/#documenting-types